src/@commonsku/styles/Thermometer.tsx

```diff
@@ -65,25 +104,26 @@ export default function Thermometer({
     const [containerWidth, setContainerWidth] = useState(0);
     const [targetWidth, setTargetWidth] = useState(0);
     const [value1Width, setValue1Width] = useState(0);
+    const [windowWidth] = useWindowSize();
 
-    const calcTargetWidth = () => {
+    const calcTargetWidth = useCallback(() => {
```

Comment:
instead of using it as a function, I think it's better to convert to useMemo and use as a value
Same for `calcVal1Width`